README.adoc

```diff
@@ -34,20 +34,34 @@ Learn how to use MicroProfile Rest Client to invoke RESTful microservices over H
 
 == What you'll learn
 
-You will learn how to build a MicroProfile Rest Client to access remote RESTful services. You will create a template interface that maps to the remote service that you want to call.
-MicroProfile Rest Client automatically generates a client instance based on what is defined and annotated in the template interface.
-Thus, you don't have to worry about all of the boilerplate code, such as setting up a client class, connecting to the remote server, or invoking the correct URI with the correct parameters.
-
-The application that you will be working with is an `inventory` service, which fetches and stores the system property information for different hosts.
-Whenever a request is made to retrieve the system properties of a particular host, the `inventory` service will create a client to invoke the `system`
-service on that host. The `system` service simulates a remote service in the application.
-
-You will instantiate the client and use it in the `inventory` service. You can choose from two different approaches, https://openliberty.io/docs/ref/general/#contexts_dependency_injection.html[Context and Dependency Injection (CDI)^] with the help of MicroProfile Config or the https://openliberty.io/blog/2018/01/31/mpRestClient.html[RestClientBuilder^] method.
-In this guide, you will explore both methods to handle scenarios for providing a valid base URL.
-
- - When the base URL of the remote service is static and known, define the default base URL in the configuration file. Inject the client with CDI method.
-
- - When the base URL is not yet known and needs to be determined during the run time, set the base URL as a variable. Build the client with the more verbose `RestClientBuilder` method.
+You will learn how to build a MicroProfile Rest Client to access remote RESTful
+services. You will create a template interface that maps to the remote service
+that you want to call. MicroProfile Rest Client automatically generates a client
+instance based on what is defined and annotated in the template interface. Thus,
+you don't have to worry about all of the boilerplate code, such as setting up a
+client class, connecting to the remote server, or invoking the correct URI with
+the correct parameters.
+
+The application that you will be working with is an `inventory` service, which
+fetches and stores the system property information for different hosts. Whenever
+a request is made to retrieve the system properties of a particular host, the
+`inventory` service will create a client to invoke the `system` service on that
+host. The `system` service simulates a remote service in the application.
+
+You will instantiate the client and use it in the `inventory` service. You can
+choose from two different approaches,
+https://openliberty.io/docs/ref/general/#contexts_dependency_injection.html[Context
+and Dependency Injection (CDI)^] with the help of MicroProfile Config or the
```

Comment:
```
https://openliberty.io/docs/ref/general/#contexts_dependency_injection.html[Context
and Dependency Injection (CDI)^]
``` 
better to be in a line because the converter may not be able to convert it correctly